tests/unit/parsers/test.manifestjson.js

```diff
@@ -96,6 +96,33 @@ describe('ManifestJSONParser', () => {
     expect(warnings[0].code).toEqual('IGNORED_APPLICATIONS_PROPERTY');
   });
 
+  it('should fail if "homepage_url" links to AMO domain names', () => {
```

Comment:
Nit, let's not quote homepage_url here, for consistency with the one that follows.